build-system/tasks/e2e/describes-e2e.js

```diff
@@ -488,8 +489,10 @@ class EndToEndFixture {
       const config = getConfig();
       const controller = await getController(config, browserName, deviceName);
       const ampDriver = new AmpDriver(controller);
+      const requestBank = new RequestBankE2E('http://localhost:8000', 'e2e');
```

Comment:
Is the 2nd param necessary?

Reply:
For now yes, I am trying to find a way to do dynamic bankId.